tests/flow/test_null_handling.py

```
@@ -18,15 +18,15 @@ def test01_create_null(self):
             query = """MATCH (a) OPTIONAL MATCH (a)-[nonexistent_edge]->(nonexistent_node) CREATE (nonexistent_node)-[:E]->(a)"""
             self.graph.query(query)
             assert(False)
-        except redis.exceptions.ResponseError:
+        except redis.ResponseError:
```

Ruff (0.15.7)

[error] `redis` may be undefined, or defined from star imports

(F405)